lib/accessly/policy/base.rb

```diff
@@ -54,27 +54,36 @@ def segment_id
         nil
       end
 
-      def grant(action, object = nil)
-        object_id = _get_object_id(object)
+      def can?(action, object = nil)
```

Comment:
Originally I just had this implemented as 
```
object_id = _get_object_id(object)
action_id = _get_action_id(action, object_id)
_can_do_action?(action, action_id, object)
```

But that clearly didn't respect any overrides made from the other apis -- Am I doing it right?

Reply:
This method still doesn't call overrides. For example, I tried this:

```rb
class User
  attr_accessor :id
end

class TestPolicy < Accessly::Policy::Base
  actions whatever: 1, okay: 2
  actions_on_objects okay:1, something: 2

  def whatever?(object = nil)
    puts "WHATEVER"
    super
  end
end

u = User.new; u.id = 123; p = TestPolicy.new(u); p.can?(:whatever)
```

I would expect to get `WHATEVER` as output because the override is doing that, but I'm not seeing that output.

I believe the implementation should be able just to call the name of the override method, since that is guaranteed to be defined even if it's not overridden:

```rb
def can?(action, object = nil)
  send("#{action}?", object)
end
```

When I do that, I see the `WHATEVER` output. And when I *don't* define the `whatever?` override, the call still works the way it's supposed to.